tests/scripts/mbed-build.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/sh
+
+# mbed-build.sh
+#
+# This file is part of mbed TLS (https://tls.mbed.org)
+#
+# Copyright (c) 2017, ARM Limited, All Rights Reserved
+#
+# Purpose
+#
+# To run test builds of the mbed OS module for all supported targets.
```

Comment:
Rather than say 'for all supported targets', can we say 'for all specified targets'?

And can we pass or set the `TARGETS` by a command line parameter?